docs/development/documentation_preview.rst

```diff
@@ -4,11 +4,12 @@
 Documentation Preview
 *********************
 
-Most of the time it's enough to build the documentation locally and see how things are going. But sometimes 
-it's nice to share our changes and get some feedback from other collaborators. 
+To preview your changes to the documentation please:
 
-To preview your changes to the documentation first make sure you have GitHub Actions activated on your fork
-(see the Actions tab). Then, there are two ways to trigger the build:
+#. Enable GitHub Actions to run on your fork (see the *Actions* tab).
```

Comment:
Where in the actions tab is this? I can't seem to see it, so this may need a little more explanation.

Reply:
@smithis7 The workflow name is `documentation-preview`. We could mention this here.

Reply:
I think it's implied we are talking about the fork (I didn't want to repeat "fork" in that sentence).

Reply:
@epassaro I cant find where to enable GitHub actions on the action tab of my fork, that was my issue

Reply:
If you can see where to enable it it's because you've already enabled it.